src/locales/de_DE/de_DE.json

```diff
@@ -203,6 +203,9 @@
         "list_item_mask": "Achten Sie darauf, dass Sie Gesichtsmasken oder andere Gegenstände, die Ihr Gesicht verdecken, abnehmen. Brillen sind zulässig",
         "title": "Wir können Ihr Gesicht nicht erkennen"
     },
+    "avc_uploading": {
```

Comment:
Please add new locales to Lokalise and the migration.md file

Reply:
Added a new commit with all the active video capture strings to MIGRATION.md now. Will check Lokalise as well, I suspect some of the locales should already be there.